webbpsf/distortion.py

```diff
@@ -63,7 +63,7 @@ def distort_image(hdulist_or_filename, ext=0, to_frame='sci', fill_value=0,
         Value used to fill in any blank space by the skewed PSF. Default = 0.
         If set to None, values outside the domain are extrapolated.
     to_frame : str
-        Type of input coordinates. 
+        Requested of output coordinate frame. 
```

Comment:
Minor: Clarify/rephrase. "Requested of" doesn't make sense...

Reply:
Right. Accidentally removed "type"